lib/sparse-stream/shared.ts

```diff
@@ -55,7 +55,7 @@ export interface SparseWritable extends NodeJS.WritableStream {
 	_write(
 		chunk: SparseStreamChunk,
 		encoding: string,
-		callback: (err?: Error | void) => void,
+		callback: (err?: Error | void | null) => void,
```

Comment:
I think this should be
```suggestion
		callback: (err?: Error | null) => void,
```
as void doesn't really make sense as a parameter value

Reply:
removed